image-builder/README.md

```diff
@@ -0,0 +1,13 @@
+Image-builder on Scaleway
+========================
+
+
+Links
+-----
+
+- [Online Labs' Documentation](https://doc.cloud.online.net)
```

Comment:
Service try-it's README.md is outdated, review the README.md file based on image-app-docker instead